hyejin/app/src/main/java/com/joocoding/android/app/githubsearch/network/ClientFactory.kt

```diff
@@ -1,17 +1,24 @@
 package com.joocoding.android.app.githubsearch.network
 
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
 import okhttp3.OkHttpClient
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 import java.util.concurrent.TimeUnit
 
-private const val BASE_URL = "https://api.github.com/"
 
+@Module
+@InstallIn(SingletonComponent::class)
 object ClientFactory {
+    private const val BASE_URL = "https://api.github.com/"
     private val okHttpClient by lazy { createOkHttpClient() }
     private val retrofit by lazy { createRetrofit() }
 
-    private fun createOkHttpClient(): OkHttpClient =
+    @Provides
+    fun createOkHttpClient(): OkHttpClient =
```

Comment:
```suggestion
    @Provides
    @Singleton
    fun createOkHttpClient(): OkHttpClient =
```

정말 싱글톤 객체를 보장하려면 어노테이션이 필요하더라구요!